system/Security/Exceptions/SecurityException.php

```diff
@@ -16,20 +16,20 @@
 
 class SecurityException extends FrameworkException implements HTTPExceptionInterface
 {
-    public static function forDisallowedAction()
+    public static function forDisallowedAction(): self
```

Comment:
This is a breaking change.
https://3v4l.org/GGS0f
Please use doc comment.

Reply:
There are good explanations.
https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#breaking-changes

Reply:
Done.